hw/hw03/hw 03 sanya1213

```diff
@@ -0,0 +1 @@
+
```

Comment:
count1 = zen.count("better")
print(count1)
count2 = zen.count("never")
print(count2)
count3 = zen.count("is")
print(count3)

Reply:
№ 3.1.2
print (str.upper(zen))